### PR TITLE
data:Nishitbaria

### PR DESCRIPTION
**Pull request by Nishitbaria**

### CODE REVIEW

File: data/Nishitbaria.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    },
+    {
+      "name": "Personal website",
+      "url": "https://nishithacks.netlify.app/",
+      "icon": "FaGlobe"
+    }, 
+    {
+      "name": "Youtube",
+      "url": "https://www.youtube.com/@Nishitbaria",
+      "icon": "FaYouTube"
```
Comment (Aadarsh805) [inline data/Nishitbaria.json:37]: Everything looks great just this youtube icon needs to change to `FaYoutube`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/Nishitbaria.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Nishit Baria",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "**Open Source** Advocate | Web-Dev | Space Lover ",
+  "tags": ["Open Source", "c++", "Java", "SQL", "Open Source "],
```
Comment (Bereky) [inline data/Nishitbaria.json:6]: Looks good. But you have used 'open source' twice in the tag parameter, you may want to update it..

File: data/Nishitbaria.json (hunk 1 above)
Comment (Nishitbaria) [reply data/Nishitbaria.json:37]: Hey Aadarsh, I solve this problem in short time

File: data/Nishitbaria.json (hunk 2 above)
Comment (Nishitbaria) [reply data/Nishitbaria.json:6]: Hey Bereky, I solve this problem in short time

**Review by LinuxGamer — CHANGES_REQUESTED** [review]
Looks good, except the changes that have already been recommended

**Pull request closed without merging**